src/rviz/CMakeLists.txt

```diff
@@ -144,7 +144,7 @@ target_link_libraries(${PROJECT_NAME}
   ${TinyXML2_LIBRARIES}
   ${X11_X11_LIB}
   ${ASSIMP_LIBRARIES}
-  ${YAMLCPP_LIBRARIES}
+  ${yaml-cpp_LIBRARIES}
```

Comment:
The variable is named YAML_CPP_LIBRARIES on Ubuntu 18.04. Is it different on Mac?

Reply:
My mistake. fixed !